runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -1067,10 +1091,10 @@ NNFW_STATUS nnfw_session::set_config(const char *key, const char *value)
 
 const onert::ir::Graph *nnfw_session::primary_subgraph()
 {
-  if (_subgraphs)
+  if (_state == State::MODEL_LOADED)
```

Comment:
Session internal state is hard-coupled with members like `_subgraphs`, which is to be removed with this PR.